src/obfuscation.spec.ts

```diff
@@ -17,6 +17,15 @@ describe('obfuscation', () => {
     });
   });
 
+  it('encodes/decodes special characters', () => {
+    const strings = ['kümmert', 'заботится', '照顾', '🤗🌸'];
+
+    strings.forEach((string) => {
+      expect(decodeBase64(encodeBase64(string))).toEqual(string);
+      expect(decodeBase64(encodeBase64(string))).toEqual(string);
```

Comment:
This only checks that encode/decode functions match within SDK, and that was true even before the fix:

<img width="166" alt="image" src="https://github.com/user-attachments/assets/fb916903-b6df-434b-beb8-534bc40220ee" />

Shall we add a couple of test case to check that decoding matches *server* encoding?
```js
expect(decodeBase64('a8O8bW1lcnQ=').toEqual('kümmert'));
```